### ISSUE TITLE
How to extract JSON list formatting (Header, Body and Footer) code from source to target site using the PnP.PowerShell Provisioning engine

### ISSUE THREAD

**Issue by astatke** [pnp/powershell#1361]
### Notice
Many bugs reported are actually related to the PnP Framework which is used behind the scenes. Consider carefully where to report an issue:

1. **Are you using ```Invoke-PnPSiteTemplate``` or ```Get-PnPSiteTemplate```**? The issue is most likely related to the Provisioning Engine. The Provisioning engine is _not_ located in the PowerShell repo. Please report the issue here: https://github.com/pnp/pnpframework/issues.
I am using Get-PnPSiteTemplate 

1. **Is the issue related to the cmdlet itself, its parameters, the syntax, or do you suspect it is the code of the cmdlet that is causing the issue?** Then please continue reporting the issue in this repo.
 I am not sure.

1. **If you think that the functionality might be related to the underlying libraries that the cmdlet is calling** (We realize that might be difficult to determine), please first double check the code of the cmdlet, which can be found here: https://github.com/pnp/powershell/tree/master/src/Commands. If related to the cmdlet, continue reporting the issue here, otherwise report the issue at https://github.com/pnp/pnpframework/issues

### Reporting an Issue or Missing Feature
Please confirm what it is that your reporting

My extracted xml file does not have the JSON code I applied for customizing the source list form's Header, Body and Footer. Is there a way how we can extract the JSON list formatting code as part of the Get-PnPSiteTemplate cmdlet?

### Expected behavior 
Please describe what output you expect to see from the PnP PowerShell Cmdlets
I was expecting to get a customized list with all the JSON code in my target site. FYI,  I opened the XML file and I don't see the JSON code there. On a side note I am using this cmdlet, Invoke-PnPSiteTemplate, against the XML file created using Get-PnPSiteTemplate cmdlet. This has created my custom list with no issues except the act that my JSON list formatting code is not there.
 
### Actual behavior 
Please describe what you see instead. Please provide samples of output or screenshots.
My custom list inside the target site does not have the Header, Body and Footer JSON formatting code.

### Steps to reproduce behavior
Please include complete script or code samples in-line or linked from [gists](https://gist.github.com/)

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name "PnP.PowerShell" -ListAvailable```)
version 1.6.44

### Which operating system/environment are you running PnP PowerShell on?
- [X ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment]
@astatke - We are now in `1.8.0` so please update the module. 
However, was it working before and not anymore? Or is it a question you're asking?

**Comment by astatke** [comment]
Thank you @veronicageek  for getting back to me on this. The issue I am having is not a bug per se. It is rather to inquire if there is a way to export a configured list form i.e. as described on this MSFT doc (https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/list-form-configuration) using the PnP.PowerShell provisioning engine.

Basically I have a custom SPO list where I added JSON formatting for the Header, Body and Footer. Next I would like to export this list as part of the site template creation process i.e. using the PnP.PowerShell cmdlet “Get-PnPSiteTemplate”. In doing so my custom JSON formatting appears to be not exported. I say this because, when I applied the exported template to a targeted site i.e. using the "Invoke-PnPSiteTemplate", my custom JSON  code did not show up in that list. If there is a way where I can be able to export such Header, Body and Footer custom JSON code and import it to a targeted site via the PnP provision engine that would be great.
